### ISSUE TITLE
GX80 - functional only for film mode

### ISSUE THREAD

**Issue by ninetto** [Rambalac/GMaster#22]
Hi, really fun/useful having this on a large-screen laptop. Unfortunately, the command for a single stills shot causes my (European) GX80 to freeze in an endless movie-record mode which can only be stopped by removing battery.  Any hope on fixing this? best regards, n.m.

**Comment by Rambalac** [comment]
Does this happen always? A bit strange that whole camera freeze.

**Comment by ninetto** [comment]
Yes, at least the 3 or 4 times I tried. When hitting the "stills camera" icon, the camera starts recording a film instead and can only be stopped by removing the battery. But the recording video seems to work fine, which is really great seeing the recorded image on a large screen.
